File: src/Other/MicroPython/singleton.py
class Registry:
    _instance = None

    def __new__(cls):
        if cls._instance is None:
            cls._instance = super().__new__(cls)
            cls._instance.count = 0
        return cls._instance


first = Registry()
second = Registry()
first.count += 1
print("same=" + ("true" if first is second else "false"))
print("count=" + str(second.count))

File: src/Web/Dart/singleton.dart
class Registry {
  Registry._();

  static final Registry _instance = Registry._();

  factory Registry() => _instance;

  int count = 0;
}

void main() {
  final first = Registry();
  final second = Registry();
  first.count += 1;
  print('same=${identical(first, second)}');
  print('count=${second.count}');
}
